Products/GenericSetup/context.py

```diff
@@ -490,7 +490,7 @@ def writeDataFile( self, filename, text, content_type, subdir=None ):
             subdir = filename[:sep]
             filename = filename[sep+1:]
 
-        if isinstance(text, unicode):
+        if six.PY2 and isinstance(text, unicode):
```

Comment:
Why not `if isinstance(text, six.text_type):` as above?

Reply:
The inconsistency is due to multiple people having worked on this changeset over time. I'll change it.